pkg/middleware/middleware_test.go

```diff
@@ -96,35 +98,66 @@ func (s *TestAuthMiddlewareSuite) TestAuthMiddlewareService() {
 	// Check that Engine() returns the router object.
 	require.NotNil(s.T(), srv.Engine())
 
-	// do some requests
-	var authtests = []struct {
-		name        string
-		urlPath     string
-		method      string
-		tokenHeader string
-		status      int
-	}{
-		{"static, no auth", "/favicon.ico", "GET", "", http.StatusOK},
-		{"health, no auth", "/api/v1/health", "GET", "", http.StatusOK},
-		{"auth_test, no auth, denied", "/api/v1/auth_test", "GET", "", http.StatusUnauthorized},
-		{"auth_test, valid header auth", "/api/v1/auth_test", "GET", "Bearer " + tokenValid, http.StatusOK},
-		{"auth_test, invalid header auth, no email claim", "/api/v1/auth_test", "GET", "Bearer " + tokenInvalidNoEmail, http.StatusUnauthorized},
-		{"auth_test, invalid header auth, expired", "/api/v1/auth_test", "GET", "Bearer " + tokenInvalidExpired, http.StatusUnauthorized},
-		{"auth_test, invalid header auth, token garbage", "/api/v1/auth_test", "GET", "Bearer " + tokenInvalidGarbage, http.StatusUnauthorized},
-		{"auth_test, invalid header auth, wrong header format", "/api/v1/auth_test", "GET", tokenValid, http.StatusUnauthorized},
-		{"auth_test, invalid header auth, bearer but no token", "/api/v1/auth_test", "GET", "Bearer ", http.StatusUnauthorized},
-	}
-	for _, tt := range authtests {
-		s.Run(tt.name, func() {
-			resp := httptest.NewRecorder()
-			req, err := http.NewRequest(tt.method, tt.urlPath, nil)
-			require.NoError(s.T(), err)
-			if tt.tokenHeader != "" {
-				req.Header.Set("Authorization", tt.tokenHeader)
-			}
-			srv.Engine().ServeHTTP(resp, req)
-			// Check the status code is what we expect.
-			assert.Equal(s.T(), tt.status, resp.Code, "request returned wrong status code: got %v want %v", resp.Code, tt.status)
-		})
-	}
+	s.Run("health check requests", func() {
+		health := &controller.Health{}
+		resp := httptest.NewRecorder()
+		req, err := http.NewRequest(http.MethodGet, "/api/v1/health", nil)
+		require.NoError(s.T(), err)
+
+		srv.Engine().ServeHTTP(resp, req)
+		// Check the status code is what we expect.
+		assert.Equal(s.T(), http.StatusOK, resp.Code, "request returned wrong status code: got %v want %v", resp.Code, http.StatusOK)
```

Comment:
No need to pass the expected and actual values as the message args. `asser.Equal()` will include it into the test failure message for you.
```suggestion
		assert.Equal(s.T(), http.StatusOK, resp.Code, "request returned wrong status code")
```